dashboard/modules/snapshot/snapshot_head.py

```diff
@@ -120,6 +129,28 @@ async def get_job_info(self):
 
         return jobs
 
+    async def get_job_submission_info(self):
+        """Info for Ray job submission.  Here a job can have 0 or many drivers."""
+
+        jobs = {}
+
+        for job_submission_id, job_data in self._job_data_client.get_all_jobs().items():
+            if job_data is not None:
+                status_info = job_data.status_info
+                entry = {
+                    "status": None if status_info is None else status_info.status,
+                    "status_message": None
+                    if status_info is None
+                    else status_info.message,
```

Comment:
to match the get & list jobs APIs, I think this should be called `message`. Can we use the same datatclass for both to ensure consistency?

Reply:
I slightly preter status_message because it's more specific; I'll update the get and list job API to have `status_message` as well.

Reply:
One consideration here is the `GET /api/jobs/status` currently returns a `message` field (I think), so it'd be a breaking change to rename it. That is ok because the API is still alpha until 1.12 but it's still preferable to avoid the breakage.